muikku-school-data-pyramus/src/main/java/fi/otavanopisto/muikku/plugins/schooldatapyramus/PyramusUserSchoolDataBridge.java

```diff
@@ -1180,6 +1181,21 @@ public boolean isActiveUser(User user) {
     return true;
   }
   
+
+  @Override
+  public BridgeResponse<List<OrganizationContactPerson>> listOrganizationContactPersonsByOrganization(
+      Long organizationId) {
+    BridgeResponse<OrganizationContactPerson[]> response = pyramusClient.responseGet(String.format("/organizations/%d/contactPerson", organizationId), OrganizationContactPerson[].class);
+    List<OrganizationContactPerson> contactPersons = null;
+    if(response.getEntity() != null) {
+      contactPersons = new ArrayList<>();
```

Comment:
Is the loop needed here or could the original list just be passed on? I'm not sure about it, probably need to find similar calls and inspect if it's been used or not.